### PR TITLE
Change of the compute  severity for Aqua JSON parser

### PR DESCRIPTION
**Pull request by SPoint42**
Modification for the compute of Aqua scan JSON import as there is a `aqua_severity` we can use. See #4466

### CODE REVIEW

**Comment by valentijnscholten** [comment]
Looks like the unit tests need an update to reflect the changes made:
```
uwsgi_1         | FAIL: test_aqua_parser_cvssv3_has_many_findings (dojo.unittests.tools.test_aqua_parser.TestAquaParser)
uwsgi_1         | ----------------------------------------------------------------------
uwsgi_1         | Traceback (most recent call last):
uwsgi_1         |   File "/app/dojo/unittests/tools/test_aqua_parser.py", line 59, in test_aqua_parser_cvssv3_has_many_findings
uwsgi_1         |     self.assertEqual(16, nb_cvssv3)
uwsgi_1         | AssertionError: 16 != 0
uwsgi_1         | 
```

**Comment by SPoint42** [comment]
> Looks like the unit tests need an update to reflect the changes made:
> 
> ```
> uwsgi_1         | FAIL: test_aqua_parser_cvssv3_has_many_findings (dojo.unittests.tools.test_aqua_parser.TestAquaParser)
> uwsgi_1         | ----------------------------------------------------------------------
> uwsgi_1         | Traceback (most recent call last):
> uwsgi_1         |   File "/app/dojo/unittests/tools/test_aqua_parser.py", line 59, in test_aqua_parser_cvssv3_has_many_findings
> uwsgi_1         |     self.assertEqual(16, nb_cvssv3)
> uwsgi_1         | AssertionError: 16 != 0
> uwsgi_1         | 
> ```

✔️

**Review by damiencarol — APPROVED** [review]

File: dojo/unittests/tools/test_aqua_parser.py [hunk 1]
```diff
@@ -57,3 +57,63 @@ def test_aqua_parser_cvssv3_has_many_findings(self):
...
+        with open("dojo/unittests/scans/aqua/vulns_with_aqua_severity.json") as testfile:
+            parser = AquaParser()
+            findings = parser.get_findings(testfile, Test())
+            nbsev = 0
+
+            for finding in findings:
+                if finding.severity == 'Info':
+                    nbsev = nbsev + 1
+
+            self.assertEqual(7, nbsev)
```
Comment (valentijnscholten) [inline dojo/unittests/tools/test_aqua_parser.py:119]: Could you merge these into 1 test case `...for_severities` and use the `.count(...)` method of the findings list to assert the result?
i.e. option 3 from https://www.geeksforgeeks.org/python-count-occurrences-element-list/

Comment (SPoint42) [reply dojo/unittests/tools/test_aqua_parser.py:119]: will do !

**Review by valentijnscholten — APPROVED** [review]

**Pull request merged**